File: DheerajKumar/problem18.cpp
//18. WAP to take input 3 numbers from the user and print the numbers in the descending order.

#include<iostream>
#include<vector>
using namespace std;

void inDecendingOrder(int number1, int number2, int number3){
    int sum = number1 + number2 + number3, 
        maximum = max(number1, max(number2, number3)),
            minimum = min(number1, min(number2, number3));

    cout<<"Decending Order -> { "<<maximum;
    cout<<", "<<sum - maximum - minimum;
    cout<<", "<<minimum<<" }";
}

int main(){
    vector<int> number(3,0);
    cout<<"\n\n";
    for(int i = 0; i < 3; i++){
        cout<<"Enter ["<<i<<"] number -> ";
        cin>>number[i];
    }
    inDecendingOrder(number[0], number[1], number[2]);
}

File: Ashish Chauhan/problems-40/problem18.java
// 18. WAP to take input 3 numbers from the user and print the numbers in the descending order.
import java.util.Scanner;

public class DescendingOrder {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Enter first number: ");
        int num1 = scanner.nextInt();
        
        System.out.print("Enter second number: ");
        int num2 = scanner.nextInt();
        
        System.out.print("Enter third number: ");
        int num3 = scanner.nextInt();
        
        int temp;
        
        if (num1 < num2) {
            temp = num1;
            num1 = num2;
            num2 = temp;
        }
        if (num1 < num3) {
            temp = num1;
            num1 = num3;
            num3 = temp;
        }
        if (num2 < num3) {
            temp = num2;
            num2 = num3;
            num3 = temp;
        }
        
        System.out.println("Numbers in descending order: " + num1 + ", " + num2 + ", " + num3);
        
        scanner.close();
    }
}
